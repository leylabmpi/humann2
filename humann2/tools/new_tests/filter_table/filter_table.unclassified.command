# basic prevalence filter
humann2_filter_table \
	     -i filter_table.input2 \
	     -L M2 \
	     -o filter_table.unclassified.output \
	     -u 50 \