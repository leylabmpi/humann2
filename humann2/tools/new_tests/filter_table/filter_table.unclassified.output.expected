# SAMPLES	S1	S2	S3	S4
M1	-	-	-	-
M2	-	-	-	-
F1	2	2	2	0
F1|S1	0	2	0	0
F1|S2	2	0	2	0
F2	3	0	0	0
F2|S1	1	0	0	0
F2|S2	1	0	0	0
F2|S3	1	0	0	0
